pages.ja/common/bc.md

```diff
@@ -0,0 +1,29 @@
+# bc
+
+> 任意の精度で計算を行える言語です。
+> `dc`も参照してください。
+> 詳細はこちら: <https://manned.org/man/bc.1>.
```

Comment:
We have a [translation](https://github.com/tldr-pages/tldr/blob/a36f018973cffa325638e4131f8ceb0336f019df/scripts/set-more-info-link.py#L25) sheet for more info. 
```suggestion
> 詳しくはこちら: <https://manned.org/man/bc.1>.
```